src/react-fontawesome.css

```diff
@@ -0,0 +1,10 @@
+.sr-only {
```

Comment:
This module should not include any CSS as that would require a build process for the end user. Is this CSS actually needed?